### PR TITLE
Chore: update pre-commit hooks

### PR DESCRIPTION
**Pull request by NripeshN**

### CODE REVIEW

File: docs/src/_static/mlx_logo.png
Comment (awni) [inline docs/src/_static/mlx_logo.png:1]: What's happening here?

Comment (awni) [reply docs/src/_static/mlx_logo.png:1]: Could you remove those diffs? I see `invalid image source` here.

Comment (NripeshN) [reply docs/src/_static/mlx_logo.png:1]: Done

**Review by awni — APPROVED** [review]
Thanks!

**Pull request merged**